README.adoc

```diff
@@ -23,9 +23,9 @@ image:https://badges.gitter.im/eclipse/microprofile-service-mesh.svg[link="https
 
 === Rationale
 
-Based on some link:https://groups.google.com/forum/#!searchin/microprofile/istio%7Csort:date/microprofile/7obnAXjt3QA/k4htskrcBwAJ [discussion], we need to investigate the relationship between MicroProfile and Service Mesh. 
+Based on some https://groups.google.com/forum/#!searchin/microprofile/istio%7Csort:date/microprofile/7obnAXjt3QA/k4htskrcBwAJ[discussion], we need to investigate the relationship between MicroProfile and Service Mesh. 
 
-Why? 
+===Why? 
```

Comment:
Think it needs a space

Reply:
ah. My Eclipse plugin auto rendered ok. Added the space. thanks @kenfinnigan